.config/s3.config.php

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Oof. Lose this! :)

Reply:
Perfect! Thanks :+1: 

I didnt know one line was so powerful ;)